packages/components/CHANGELOG-FIGMA-COMPONENTS.md

```diff
@@ -1,4 +1,23 @@
 # [HDS Product - Components](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components?type=design&node-id=6790-10926&mode=design&t=Ps0aMGZ6F3z7bAJ4-0)
+## December 20th, 2024
+
+`IconTile` - Visual updates made to the component, including removing the border and updating colors for improved contrast and alignment.
```

Comment:
I might add "visual" in here because alignment in this context sounds like a layout term.

```suggestion
`IconTile` - Visual updates made to the component, including removing the border and updating colors for improved contrast and visual alignment.
```